<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-12-27 15:31:41.516 ***</b>
=ERROR REPORT==== 27-Dec-2024::15:31:41.516761 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.245414.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1973}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1934}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-12-27 15:31:41.517 ***</b>
=CRASH REPORT==== 27-Dec-2024::15:31:41.517073 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.245420.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2426)
    ancestors: [<0.245414.0>]
    message_queue_len: 0
    messages: []
    links: [<0.245414.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 987
    stack_size: 29
    reductions: 594
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-12-27 15:32:21.665 ***</b>
=ERROR REPORT==== 27-Dec-2024::15:32:21.665288 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.249764.0&gt;,
                               {#Ref&lt;0.2273491840.2396520449.194134&gt;,1004185,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.2273491840.2396520449.194134&gt;,1004185,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-12-27 15:32:21.665 ***</b>
=CRASH REPORT==== 27-Dec-2024::15:32:21.665578 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.249785.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.2273491840.2396520449.194134&gt;,1004185,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [&lt;0.249764.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 410
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2024-12-27 15:37:05.532 ***</b>
=CRASH REPORT==== 27-Dec-2024::15:37:05.532080 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.255596.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.2273491840.2398093313.10937&gt;,5620,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1694)
    ancestors: [&lt;0.255594.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28123
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-12-27 15:37:14.783 ***</b>
=ERROR REPORT==== 27-Dec-2024::15:37:14.783077 ===
** Generic server <0.256886.0> terminating 
** Last message in was {nodedown,'peer_down_crash-370117-11899@414d51cabcc6'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-370117-11899",
                              peer_down => crash},
                            'peer_down_crash-370117-11899@414d51cabcc6',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,50,47,98,105,110,
                             47,101,114,108],
                            ["-sname","peer_down_crash-370117-11899",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQDQxNGQ1MWNhYmNjNgAD63YAAAAAZ2638A=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-370117-11899@414d51cabcc6'}},
                            {<0.256861.0>,
                             #Ref<0.2273491840.2390491140.223976>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-370117-11899@414d51cabcc6'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2024-12-27 15:37:14.783 ***</b>
=CRASH REPORT==== 27-Dec-2024::15:37:14.783466 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256886.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-370117-11899@414d51cabcc6'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [<0.256861.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1482
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-12-27 15:37:15.966 ***</b>
=WARNING REPORT==== 27-Dec-2024::15:37:14.964582 ===
Process <0.92.0> on node 'peer_SUITE-shutdown_stop-521794-11899@414d51cabcc6' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-12-27 15:37:16.321 ***</b>
=WARNING REPORT==== 27-Dec-2024::15:37:16.321584 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-370181-11899@414d51cabcc6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-12-27 15:37:16.440 ***</b>
=WARNING REPORT==== 27-Dec-2024::15:37:16.440639 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-450020-11899@414d51cabcc6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-12-27 15:37:32.205 ***</b>
=WARNING REPORT==== 27-Dec-2024::15:37:31.203767 ===
Process <0.89.0> on node 'peer_SUITE-shutdown_stop-370341-11899@414d51cabcc6' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-12-27 15:37:32.557 ***</b>
=WARNING REPORT==== 27-Dec-2024::15:37:32.557671 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-522498-11899@414d51cabcc6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-12-27 15:37:32.570 ***</b>
=WARNING REPORT==== 27-Dec-2024::15:37:32.570574 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-522626-11899@414d51cabcc6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-12-27 15:37:34.783 ***</b>
=ERROR REPORT==== 27-Dec-2024::15:37:34.783557 ===
** Generic server <0.257240.0> terminating 
** Last message in was {'EXIT',<0.257222.0>,
                               {#Ref<0.2273491840.2390491140.225857>,1480748,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-370533-11899",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-370533-11899@414d51cabcc6',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,50,47,98,105,110,47,101,114,108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-370533-11899",
                                     "-user","peer"],
                                    #Port<0.3124>,undefined,<<>>,running,
                                    {<0.257222.0>,
                                     #Ref<0.2273491840.2390491140.226088>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.2273491840.2390491140.225857>,1480748,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-12-27 15:37:34.784 ***</b>
=CRASH REPORT==== 27-Dec-2024::15:37:34.783909 ===
  crasher:
    initial call: peer:init/1
    pid: <0.257240.0>
    registered_name: []
    exception exit: {#Ref<0.2273491840.2390491140.225857>,1480748,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.257222.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3124>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4634
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-12-27 15:37:35.473 ***</b>
=ERROR REPORT==== 27-Dec-2024::15:37:35.473468 ===
** Generic server <0.257267.0> terminating 
** Last message in was {'EXIT',<0.257221.0>,
                           {#Ref<0.2273491840.2390491140.225855>,2168505,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "AKQOUFPHXOSSMVTKVKNH","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@414d51cabcc6.2024-12-27_15.21.35/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-450788-11899",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@414d51cabcc6.2024-12-27_15.21.35/make_test_dir.stdlib_test.logs/run.2024-12-27_15.22.11/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-450788-11899",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5.6/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-450788-11899@414d51cabcc6',
                                    "/usr/local/lib/erlang-26.2.5.6/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-450788-11899",
                                     "-setcookie","AKQOUFPHXOSSMVTKVKNH",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@414d51cabcc6.2024-12-27_15.21.35/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-450788-11899",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@414d51cabcc6.2024-12-27_15.21.35/make_test_dir.stdlib_test.logs/run.2024-12-27_15.22.11/log_private/",
                                     "-user","peer"],
                                    #Port<0.3152>,undefined,<<>>,running,
                                    {<0.257221.0>,
                                     #Ref<0.2273491840.2390753283.41258>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.2273491840.2390491140.225855>,2168505,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-12-27 15:37:35.474 ***</b>
=CRASH REPORT==== 27-Dec-2024::15:37:35.473937 ===
  crasher:
    initial call: peer:init/1
    pid: <0.257267.0>
    registered_name: []
    exception exit: {#Ref<0.2273491840.2390491140.225855>,2168505,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.257221.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3152>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4321
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-12-27 15:37:35.873 ***</b>
=WARNING REPORT==== 27-Dec-2024::15:37:34.871680 ===
Process <0.89.0> on node 'peer_SUITE-shutdown_stop-450724-11899@414d51cabcc6' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-12-27 15:37:35.920 ***</b>
=WARNING REPORT==== 27-Dec-2024::15:37:35.920692 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-450468-11899@414d51cabcc6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-12-27 15:37:36.261 ***</b>
=WARNING REPORT==== 27-Dec-2024::15:37:36.261588 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-370565-11899@414d51cabcc6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-12-27 15:37:36.300 ***</b>
=WARNING REPORT==== 27-Dec-2024::15:37:36.300586 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-450756-11899@414d51cabcc6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-12-27 15:58:54.789 ***</b>
=ERROR REPORT==== 27-Dec-2024::15:58:54.789113 ===
** Generic server <0.334156.0> terminating 
** Last message in was {'EXIT',<0.334154.0>,
                               {#Ref<0.2273491840.2429026305.236901>,350704,
                                [{child1,<0.334163.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.334156.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.334162.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458514],
                            1,0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.2273491840.2429026305.236901>,350704,
    [{child1,<0.334163.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-12-27 15:58:54.789 ***</b>
=ERROR REPORT==== 27-Dec-2024::15:58:54.789113 ===
** Generic server <0.334157.0> terminating 
** Last message in was {'EXIT',<0.334154.0>,
                               {#Ref<0.2273491840.2429026305.236901>,350704,
                                [{child1,<0.334163.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.334157.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.334163.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458514],
                            1,0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.2273491840.2429026305.236901>,350704,
    [{child1,<0.334163.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-12-27 15:58:54.789 ***</b>
=CRASH REPORT==== 27-Dec-2024::15:58:54.789511 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.334156.0>
    registered_name: []
    exception exit: {#Ref<0.2273491840.2429026305.236901>,350704,
                     [{child1,<0.334163.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.334154.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1010
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-12-27 15:58:54.789 ***</b>
=CRASH REPORT==== 27-Dec-2024::15:58:54.789661 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.334157.0>
    registered_name: []
    exception exit: {#Ref<0.2273491840.2429026305.236901>,350704,
                     [{child1,<0.334163.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.334154.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1027
  neighbours:


</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Fri Dec 27 2024 15:22:11<!--/date--><br />
</div>
</center>
</body>
</html>
